README.md

```diff
@@ -1,14 +1,122 @@
+
+### Database... 
+Zebedee isn't backed by a database instead it uses a file system to store json files on disk ***. As a result it 
+requires a specific directory structure in order to function correctly.
+
+***
+*** _We know this is a terrible idea - but in our defence this is a legacy hangover and we are actively working 
+towards deprecating it._
+***
+
+To save yourself some pain you can use the [dp-zebedee-utils/content][3] tool to create the required directory 
+structure and populate the CMS with some "default content" - follow the steps in the [README][3] before going any further.
+ 
+Once the script has run successfully copy the generated script from `dp-zebedee-utils/content/generated/run-cms.sh` into
+the root dir of your Zebedee project. This bash script compiles and runs Zebedee setting typical dev default 
+configuration and uses the content / directory structure generated by dp-zebedee-utils.
+
+You may be required to make the bash script an executable before you can run it. If so run:
+
+````bash
+sudo chmod +x run-cms.sh
+<Enter your password when prompted>
+````  
+
+### Running the publishing stack
+In order to use Zebedee you will need to have the following other project running:
+- Florence
+- Babbage
+- Sixteens
+- dp-compose
+
+Follow the steps in the README of each project.
+
+#### Running zebedee 
+```bash
+./run-cms.sh
+```
+
+Assuming Zebedee has started without error head to [Florence login][4] and login with the default account:
+```
+email: florence@magicroundabout.ons.gov.uk
+password: Doug4l
+```
+- If it's the first time logging in you will be prompted to change the password for that user.
+- On the home screen create a new collection.
+- Click `Create/edit` on the Collection Details screen.
+
+If everything is working correctly you should now see the the ONS website displayed in the right hand pane. 
+_Congratulations_ :tada:! Advanced to GO collect £200 :dollar:
+
+Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
+
+#### Service authentication with Zebedee
+
+1) Login to florence using: `curl -X POST -d '{"email":"florence@magicroundabout.ons.gov.uk","password":"<your password>"}' http://localhost:8082/login`
+2) Make a note of the `access_token` that gets returned in the headers
+3) Create an admin service key: `curl -X POST http://localhost:8082/service -H "X-Florence-Token: <access_token>" -d '{"id":"admin"}'`
+4) Make a note of the service token that gets returned in the response body
+5) Set the environment variable:
+`export SERVICE_AUTH_TOKEN=<YOUR_SERVICE_TOKEN>` replacing the token with that one you got in step 4
+6) Restart zebedee and authenticating services
+
+
+[1]: https://github.com/ONSdigital/babbage
+[2]: https://github.com/ONSdigital/florence
+[3]: https://github.com/ONSdigital/dp-zebedee-utils/tree/master/content
+[4]: http://localhost:8081/florence/login
+[5]: https://github.com/ONSdigital/sixteens
+[6]: https://github.com/ONSdigital/dp-compose
+>>>>>>> develop
```

Comment:
Not gonna fail the PR but we can probably remove this merge conflict from the readme :)